app/assets/stylesheets/navbar.scss

```diff
@@ -0,0 +1,9 @@
+.wight_Log_Sign {
```

Comment:
@SemyzSerhiy you use weird class naming. It is some mix of camel case and snake case. Let's use `wight-log-sign` style for css naming